crates/swc_ecma_transforms_module/src/util.rs

```diff
@@ -161,6 +161,17 @@ pub struct Scope {
 }
 
 impl Scope {
+    /// insert exported bindings without duplication
+    pub(crate) fn insert_exported_bindings(&mut self, ident: &Ident) {
+        let bindings = self
+            .exported_bindings
+            .entry((ident.sym.clone(), ident.span.ctxt()))
+            .or_default();
+        if !bindings.contains(&(ident.sym.clone(), ident.span.ctxt())) {
```

Comment:
Please use `ident.to_id()` instead

Reply:
`(ident.sym.clone(), ident.span.ctxt())` is old version of `ident.to_id()`

Reply:
ok